generate_workspace/src/main/java/com/google/devtools/build/workspace/maven/Resolver.java

```diff
@@ -137,7 +137,7 @@ public void resolveArtifact(String artifactCoord) {
     deps.put(rule.name(), rule);  // add the artifact rule to the workspace
     Model model = modelResolver.getEffectiveModel(modelSource);
     if (model != null) {
-      traverseDeps(model, Sets.newHashSet(), rule);
+      traverseDeps(model, Sets.newHashSet(JavaScopes.COMPILE), Sets.newHashSet(), rule);
```

Comment:
Small nit. Can you extract the set of valid scopes into a global variable? For example:

` private static final Set<String> VALID_SCOPES = Sets.newHashSet(JavaScopes.COMPILE);`

Slightly more readable.

Reply:
Done - although this line should actually use an empty set, which I updated.